src/Commands/ScaffoldExtensionCommand.php

```diff
@@ -62,17 +62,25 @@ public function runScaffoldExtensionsJob(string $site_info = '', $job_name = '')
         $site = $this->getSite($site_id);
         $env = $site->getEnvironments()->get($site_env);
 
+        // Bail if getSiteById doesn't exist.
+        if (! method_exists($this, 'getSiteByID')) {
+            $this->log()->error('This command requires Terminus 3.2.0 or later. You appear to be running an earlier version of Terminus. Please update Terminus with terminus self:update and try again.');
```

Comment:
is terminus self:update the only correct way to use Terminus now? what about homebrew?
```suggestion
            $this->log()->error('This command requires Terminus 3.2.0 or later. You appear to be running an earlier version of Terminus. Please update Terminus and try again.');
```

Reply:
Good point, I can just say "update terminus" and be more general.